explanation.md

```diff
@@ -1,74 +1,52 @@
-[Version française](./explanation_fr.md)
+[Lire en version française](./explanation_fr.md)
 
 # Methodology
 
-Graphgenerator can recreate the network of interactions of Twitter accounts on a given hashtag or word (or group of words) 
-given. 
+Graphgenerator can recreate the network of interactions of Twitter accounts on a given hashtag or word (or group of words) given. 
```

Comment:
Graphgenerator can recreate the network of interactions of Twitter accounts on a given hashtag or word (or group of words).